staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion/conversion_test.go

```diff
@@ -686,11 +686,17 @@ func expectConversionFailureMessage(id, message string) func(t *testing.T, ctc *
 
 		for _, verb := range []string{"get", "list", "create", "update", "patch", "delete", "deletecollection"} {
 			t.Run(verb, func(t *testing.T) {
+				lastRV := objv1beta2.GetResourceVersion()
 				switch verb {
 				case "get":
-					_, err = clients["v1beta2"].Get(context.TODO(), obj.GetName(), metav1.GetOptions{})
+					var get *unstructured.Unstructured
+					get, err = clients["v1beta2"].Get(context.TODO(), obj.GetName(), metav1.GetOptions{})
+					if get != nil {
+						lastRV = get.GetResourceVersion()
+					}
 				case "list":
-					_, err = clients["v1beta2"].List(context.TODO(), metav1.ListOptions{})
+					// Use ResourceVersionMatchExact to serve request from storage while watch cache is not initialized.
```

Comment:
I think we need much better explanation here. Something like:

With ResilientWatchcCacheInitialization feature, List requests are rejected with 429 if watchcache is not initialized. However, in some of these tests that install faulty converter webhook, watchcache will never initialize by definition (as list will never succeed due to faulty converter webook). In such case, the returned error will differ from the one returned from the etcd, so we need to force the request to go to etcd.

Reply:
Done